File: go/treasure-map.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	row1 := []string{"⬜️", "️⬜️", "️⬜️"}
	row2 := []string{"⬜️", "⬜️", "️⬜️"}
	row3 := []string{"⬜️️", "⬜️️", "⬜️️"}

	matrix := [][]string{
		row1,
		row2,
		row3,
	}

	fmt.Print("Where do you want to put the treasure? ")
	var position string
	fmt.Scanln(&position)

  column_idx, err := strconv.Atoi(string(position[0]))
  if err != nil {
    panic(err)
  }

  row_idx, err := strconv.Atoi(string(position[1]))
  if err != nil {
    panic(err)
  }

  matrix[row_idx-1][column_idx-1] = "X"

  fmt.Println(matrix)
}

File: python/treasure-map.py
row1 = ["⬜️","️⬜️","️⬜️"]
row2 = ["⬜️","⬜️","️⬜️"]
row3 = ["⬜️️","⬜️️","⬜️️"]
map = [row1, row2, row3]
print(f"{row1}\n{row2}\n{row3}")
position = input("Where do you want to put the treasure? ")


column_idx = int(position[0]) - 1
row_idx = int(position[1]) - 1
map[row_idx][column_idx] = 'X'

print(f"{row1}\n{row2}\n{row3}")